.gitignore

```diff
@@ -20,6 +20,7 @@ utils/tkey_exchange/sgx_tkey_exchange_t.*
 utils/tkey_exchange/sgx_tkey_exchange_u.*
```

Comment:
"
Add log system based on log4cplus

Store log files for each service separately in the runtime folder: /var/run/ehsm

The maximum capacity of a single log file is 10MB and each service can reserve
up to 5 log files. If the log number exceeds the maximum capacity, the oldest
log will be overwritten automatically.

test: passed all the unit test
Signed-off-by: hongjingmao <hongjing.mao@intel.com>
"